packages/web-integration/tests/ai/web/playwright/image-prompt.spec.ts

```diff
@@ -0,0 +1,65 @@
+import path from 'node:path';
```

Comment:
This test should be placed in the tests/ai directory. Currently, the difference between calling ai and not calling ai will be distinguished